tests/datamodules/test_mmflood.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
For testing the datamodule, we usually add a config.yaml file in `tests/conf`and then test the datamodule directlry with one of the trainers. So the steps are:
- create a mlflood.yaml file with the specific parameters
- add this yaml file to the list of files to be tested in `tests/trainers/test_segmentation`

then this whole datamodule test file is should not be necessary